File: 01 Introdução/1847 Bem-vindos e Bem-vindas ao Inverno/1847.py
A, B, C = [int(x) for x in input().split(' ')]

if (A > B and C >= B):
    print(":)")
elif (B > A and B >= C):
    print(":(")
elif (C > B > A and C-B < B-A):
    print(":(")
elif (C > B > A and C-B >= B-A):
    print(":)")
elif (A > B > C and B-C < A-B):
    print(":)")
elif (A > B > C and B-C >= A-B):
    print(":(")
elif (A == B and C > B):
    print(":)")
else:
    print(":(")

# https://www.beecrowd.com.br/judge/pt/problems/view/1847
# Ranking 00906º, Time 0.021, Submiss Date 01/04/2023 19:11:17 (Python 3.9)

File: 01 Introdução/1847 Bem-vindos e Bem-vindas ao Inverno/1847.rb
A, B, C = gets.split.map(&:to_i)

if A > B and C >= B
  puts ":)"
elsif B > A and B >= C
  puts ":("
elsif C > B and B > A and C-B < B-A
  puts ":("
elsif C > B and B > A and C-B >= B-A
  puts ":)"
elsif A > B and B > C and B-C < A-B
  puts ":)"
elsif A > B and B > C and B-C >= A-B
  puts ":("
elsif A == B and C > B
  puts ":)"
else
  puts ":("
end

# https://www.beecrowd.com.br/judge/pt/problems/view/1847
# Ranking 00018º, Time 0.128, Submiss Date 01/04/2023 19:35:26 (Ruby)